src/Chapter2.hs

```diff
@@ -726,8 +750,11 @@ value of the element itself
 
 🕯 HINT: Use combination of 'map' and 'replicate'
 -}
+-- Assuming Negative numbres will not be present in the Input Array
+-- If they appear, they will be ignored
```

Comment:
Negative numbers are not a problem, since `replicate` returns empty list on negative numbers. It's not clear how to repeat the number `-3` time :slightly_smiling_face: